addons/website/views/snippets/s_big_number.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_big_number" name="Big number">
+    <section class="s_big_number o_half_screen_height o_cc o_cc5 pt104 pb104" style="text-align: center;" data-oe-shape-data="{'shape':'web_editor/Rainy/09_001','flip':[],'showOnMobile':false}">
```

Comment:
@chgo-odoo Don't use text-align on a section. This can't be done with Website Builder. If I'm right you did that in s_discovery as well. You can use this PR to resolve the situation there at the same time.

Reply:
Indeed, I know it's not doable with the editor but I was sure I experienced a case in which the editor added the class on the section 🤔 

Still, cannot reproduce so i'll put in on the title as you said